plugins/tasks/concord/src/main/java/com/walmartlabs/concord/client/SecretsTaskParams.java

```diff
@@ -112,13 +112,15 @@ static class CreateParams extends SecretsTaskParams {
         static final String GENERATE_PASSWORD_KEY = "generatePassword"; // NOSONAR
         static final String VISIBILITY_KEY = "visibility";
         static final String PROJECT_NAME_KEY = "project";
+        static final String PROJECT_NAMES_KEY = "projects";
+        static final String PROJECT_IDS_KEY = "projectIds";
 
         private CreateParams(Variables variables) {
             super(variables);
         }
 
         public SecretEntry.TypeEnum secretType() {
-            String type = variables.getString(Constants.Multipart.TYPE, SecretEntry.TypeEnum.DATA.toString());
+            String type = variables.assertString(Constants.Multipart.TYPE);
```

Comment:
looks like breaking change. before this changes `secretType` was optional with `DATA` type as default.

Reply:
Updated